cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -29,4 +31,28 @@ public boolean install(boolean silent) {
     return super.install(silent);
   }
 
-}
+  @Override
+  protected void postInstall() {
+
+    for (PluginDescriptor plugin : super.getConfiguredPlugins()) {
+
+      Path mavenPlugin = this.context.getSoftwarePath().resolve(this.tool).resolve(createPluginPath(plugin.getName()));
+      this.context.getFileAccess().download(plugin.getUrl(), mavenPlugin);
+      this.context.success("Successfully added {} to {}", plugin.getName(), mavenPlugin.toString());
+
+      super.postInstall();
+    }
```

Comment:
```suggestion
    }
    super.postInstall();
```

Should this rather be outside the loop?